### PR TITLE
Make UI vscode styled and refactor webview js

### PR DESCRIPTION
**Pull request by GrahamSH-LLK**
- Restyles UI to fit in with vscode
image
- Refactors the webview js
	- Previously, this used `innerHTML` to create the content and created eventListeners in another function based on DOM attributes. This refactors to create through `document.createElement`, giving access to the elements on creation so eventListeners can be created there (and version info captured). This also means that a malicious (or just broken?) vendordep can't inject any html into the sidebar and break it 
- Removes vestigial stuff from the original dep view PR
	- There were some unused CSS files and noop commands from the example extension I assume this was built on.

~~TODO: figure out how to make the deps bundled (probably requires bundling main.js too?)~~ done

### CODE REVIEW

**Review by Gold856 — CHANGES_REQUESTED** [review]
Minor formatting thing: I looked through the repo and determined this set of Prettier settings is the most consistent with the existing code. I'm considering doing a formatting job, so for consistency, these are the formatting settings you should use.
```json
{
    "singleQuote": true,
    "endOfLine": "crlf",
    "trailingComma": "es5"
}
```

**Comment by jasondaming** [comment]
Can you explain more what you are trying to do with "figure out how to make the deps bundled"?

**Comment by GrahamSH-LLK** [comment]
> Can you explain more what you are trying to do with "figure out how to make the deps bundled"?

 node_modules don’t exist in the built vsix, so the way I’m importing vscode-elements currently (uri from there) won’t work. I think I can get webpack to load the dependencies I need using raw-loader and inline as a script element. I’m open to other ideas, though!

**Review by Gold856 — APPROVED** [review]
Looks good to me

**Comment by PeterJohnson** [comment]
Needs conflicts resolved.

**Comment by PeterJohnson** [comment]
Build is failing

File: vscode-wpilib/package.json [hunk 1]
```diff
@@ -54,19 +54,20 @@
         "onCommand:wpilibcore.openApiDocumentation",
         "onCommand:wpilibcore.getProjectInformation",
         "onCommand:wpilibcore.runGradleClean",
+        "onCommand:wpilib.refreshVendordeps"
```
Comment (sciencewhiz) [inline vscode-wpilib/package.json:57]: needs a comma

File: vscode-wpilib/media/codicon.ttf
Comment (sciencewhiz) [inline vscode-wpilib/media/codicon.ttf:1]: This appears to be licensed as CC BY 4.0 which needs attribution. https://github.com/microsoft/vscode-codicons?tab=readme-ov-file#legal-notices

**Review by PeterJohnson — APPROVED** [review]

**Pull request merged**